From 56a254e3d71803aa94f768979ddc94f2572c3192 Mon Sep 17 00:00:00 2001
From: Frank Li <Frank.Li@freescale.com>
Date: Wed, 20 Jul 2011 14:52:21 +0800
Subject: [PATCH 0273/2463] ENGR00153373-2 config add ocotp to default config

Add ocotp to default config
Add watch dog to default config
Add debugfs and clk_debug to default config

Signed-off-by: Frank Li <Frank.Li@freescale.com>
---
 arch/arm/configs/imx6_defconfig |   21 +++++++++++++++++----
 1 files changed, 17 insertions(+), 4 deletions(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index 00508ae..5d68310 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux/arm 2.6.38 Kernel Configuration
-# Thu Jul 14 19:45:47 2011
+# Wed Jul 20 15:15:54 2011
 #
 CONFIG_ARM=y
 CONFIG_HAVE_PWM=y
@@ -131,6 +131,7 @@ CONFIG_HAVE_DMA_API_DEBUG=y
 #
 # GCOV-based kernel profiling
 #
+# CONFIG_GCOV_KERNEL is not set
 CONFIG_HAVE_GENERIC_DMA_COHERENT=y
 CONFIG_SLABINFO=y
 CONFIG_RT_MUTEXES=y
@@ -253,6 +254,8 @@ CONFIG_ARCH_MXC=y
 CONFIG_GPIO_PCA953X=y
 CONFIG_IMX_HAVE_PLATFORM_FEC=y
 CONFIG_IMX_HAVE_PLATFORM_FSL_USB2_UDC=y
+CONFIG_IMX_HAVE_PLATFORM_IMX2_WDT=y
+CONFIG_IMX_HAVE_PLATFORM_IMX_SNVS_RTC=y
 CONFIG_IMX_HAVE_PLATFORM_IMX_I2C=y
 CONFIG_IMX_HAVE_PLATFORM_IMX_UART=y
 CONFIG_IMX_HAVE_PLATFORM_MXC_EHCI=y
@@ -262,11 +265,13 @@ CONFIG_IMX_HAVE_PLATFORM_SPI_IMX=y
 CONFIG_IMX_HAVE_PLATFORM_IMX_IPUV3=y
 CONFIG_IMX_HAVE_PLATFORM_IMX_VPU=y
 CONFIG_IMX_HAVE_PLATFORM_AHCI=y
+CONFIG_IMX_HAVE_PLATFORM_IMX_OCOTP=y
 CONFIG_IMX_HAVE_PLATFORM_LDB=y
 CONFIG_IMX_HAVE_PLATFORM_VIV_GPU=y
 CONFIG_IMX_HAVE_PLATFORM_IMX_ANATOP_THERMAL=y
 CONFIG_IMX_HAVE_PLATFORM_FSL_OTG=y
 CONFIG_IMX_HAVE_PLATFORM_FSL_USB_WAKEUP=y
+CONFIG_IMX_HAVE_PLATFORM_IMX_PM=y
 
 #
 # Freescale MXC Implementations
@@ -293,6 +298,7 @@ CONFIG_MXC_PWM=y
 CONFIG_ARCH_MXC_IOMUX_V3=y
 CONFIG_IRAM_ALLOC=y
 CONFIG_DMA_ZONE_SIZE=96
+CONFIG_CLK_DEBUG=y
 
 #
 # System MMU
@@ -838,6 +844,7 @@ CONFIG_SMSC911X=y
 # CONFIG_IBM_NEW_EMAC_MAL_CLR_ICINTSTAT is not set
 # CONFIG_IBM_NEW_EMAC_MAL_COMMON_ERR is not set
 # CONFIG_B44 is not set
+# CONFIG_KS8842 is not set
 # CONFIG_KS8851 is not set
 # CONFIG_KS8851_MLL is not set
 CONFIG_FEC=y
@@ -998,6 +1005,7 @@ CONFIG_UNIX98_PTYS=y
 CONFIG_LEGACY_PTYS=y
 CONFIG_LEGACY_PTY_COUNT=256
 # CONFIG_TTY_PRINTK is not set
+CONFIG_FSL_OTP=y
 # CONFIG_HVC_DCC is not set
 # CONFIG_IPMI_HANDLER is not set
 CONFIG_HW_RANDOM=y
@@ -1006,7 +1014,6 @@ CONFIG_HW_RANDOM=y
 # CONFIG_RAW_DRIVER is not set
 # CONFIG_TCG_TPM is not set
 # CONFIG_RAMOOPS is not set
-CONFIG_MXC_IIM=y
 CONFIG_I2C=y
 CONFIG_I2C_BOARDINFO=y
 CONFIG_I2C_COMPAT=y
@@ -1122,6 +1129,7 @@ CONFIG_WATCHDOG_NOWAYOUT=y
 # CONFIG_SOFT_WATCHDOG is not set
 # CONFIG_MPCORE_WATCHDOG is not set
 # CONFIG_MAX63XX_WATCHDOG is not set
+CONFIG_IMX2_WDT=y
 
 #
 # USB-based Watchdog Cards
@@ -1307,6 +1315,7 @@ CONFIG_FB_MXC_LDB=y
 # CONFIG_FB_MXC_CH7026 is not set
 # CONFIG_FB_MXC_TVOUT_CH7024 is not set
 # CONFIG_FB_MXC_ASYNC_PANEL is not set
+# CONFIG_FB_MXC_EINK_PANEL is not set
 # CONFIG_FB_MXC_ELCDIF_FB is not set
 
 #
@@ -1605,10 +1614,10 @@ CONFIG_RTC_INTF_DEV_UIE_EMUL=y
 # CONFIG_RTC_DRV_MSM6242 is not set
 # CONFIG_RTC_MXC is not set
 # CONFIG_RTC_DRV_MXC_V2 is not set
+CONFIG_RTC_DRV_SNVS=y
 # CONFIG_RTC_DRV_BQ4802 is not set
 # CONFIG_RTC_DRV_RP5C01 is not set
 # CONFIG_RTC_DRV_V3020 is not set
-CONFIG_RTC_DRV_SNVS=y
 
 #
 # on-CPU RTC drivers
@@ -1730,7 +1739,9 @@ CONFIG_EXT4_FS_XATTR=y
 # CONFIG_EXT4_FS_SECURITY is not set
 # CONFIG_EXT4_DEBUG is not set
 CONFIG_JBD=y
+# CONFIG_JBD_DEBUG is not set
 CONFIG_JBD2=y
+# CONFIG_JBD2_DEBUG is not set
 CONFIG_FS_MBCACHE=y
 # CONFIG_REISERFS_FS is not set
 # CONFIG_JFS_FS is not set
@@ -1907,7 +1918,7 @@ CONFIG_FRAME_WARN=1024
 # CONFIG_MAGIC_SYSRQ is not set
 # CONFIG_STRIP_ASM_SYMS is not set
 # CONFIG_UNUSED_SYMBOLS is not set
-# CONFIG_DEBUG_FS is not set
+CONFIG_DEBUG_FS=y
 # CONFIG_HEADERS_CHECK is not set
 # CONFIG_DEBUG_KERNEL is not set
 # CONFIG_HARDLOCKUP_DETECTOR is not set
@@ -1918,6 +1929,7 @@ CONFIG_BKL=y
 CONFIG_DEBUG_BUGVERBOSE=y
 # CONFIG_DEBUG_MEMORY_INIT is not set
 # CONFIG_RCU_CPU_STALL_DETECTOR is not set
+# CONFIG_LKDTM is not set
 CONFIG_SYSCTL_SYSCALL_CHECK=y
 CONFIG_HAVE_FUNCTION_TRACER=y
 CONFIG_HAVE_FUNCTION_GRAPH_TRACER=y
@@ -1926,6 +1938,7 @@ CONFIG_HAVE_FTRACE_MCOUNT_RECORD=y
 CONFIG_HAVE_C_RECORDMCOUNT=y
 CONFIG_TRACING_SUPPORT=y
 # CONFIG_FTRACE is not set
+# CONFIG_DYNAMIC_DEBUG is not set
 # CONFIG_DMA_API_DEBUG is not set
 # CONFIG_ATOMIC64_SELFTEST is not set
 # CONFIG_SAMPLES is not set
-- 
1.7.7.4

